File: Code/Websites/DanpheEMR/Services/Accounting/DTOs/AccountTenantSectionMapDTO.cs
using DanpheEMR.ServerModel;
using System.Collections.Generic;

namespace DanpheEMR.Services.Accounting.DTOs
{
    public class AccountTenantSectionMapDTO
    {
        public string HospitalName { get; set; }
        public string HospitalShortCode { get; set; }
        public string SectionName { get; set; }
        public string SectionCode { get; set; }
        public bool IsActive { get; set; }
    }
    public class AccountTenantMapWrapperDTO
    {
        public List<AccountTenantSectionMapDTO> Maps { get; set; }
        public List<AccSectionModel> SectionList { get; set; }
    }

    public class AddAccountTenantPost_DTO
    {
        public string HospitalName { get; set; }
        public string HospitalShortCode { get; set; }
        public string SectionIds { get; set; }
    }
}

File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/accounting/shared/DTOs/account-tenant-section-map.dto.ts
import { SectionModel } from "../../settings/shared/section.model";

export class AccountTenantSectionMap_DTO {
    public HospitalName: string = "";
    public HospitalShortCode: string = "";
    public SectionName: string = "";
    public SectionCode: string = "";
    public IsActive: boolean = true;
}

export class AccountTenantMapWrapper_DTO {
    public Maps: Array<AccountTenantSectionMap_DTO> = new Array<AccountTenantSectionMap_DTO>();
    public SectionList: Array<SectionModel> = new Array<SectionModel>();
}

export class AddAccountTenantPost_DTO {
    public HospitalName: string = "";
    public HospitalShortCode: string = "";
    public SectionIds: string = "";
}